src/tracer.ts

```diff
@@ -88,6 +144,11 @@ export class Tracer {
                 options.references = [childOf];
             }
             delete(options.childOf);
+        } else if (!options.references) {
```

Comment:
Java and Python have an "ignore active span" option to prevent this automatic reference.  Any chance you'd want to follow suite?

Reply:
Yes, this should